packages/react-components/src/organisms/EventCard.tsx

```diff
@@ -1,19 +1,97 @@
 import React, { ComponentProps } from 'react';
 import { EventResponse } from '@asap-hub/model';
+import { isAfter, isBefore, subMinutes } from 'date-fns';
 
 import { ToastCard, TagList, EventInfo } from '../molecules';
+import { Link } from '../atoms';
+
+const EVENT_CONSIDERED_IN_PROGRESS_BEFORE_MINUTES: number = 5;
```

Comment:
This should be in model and needs to also be used for "This event is currently happening" in `JoinEvent`